src/Forms/GridField/GridField.php

```diff
@@ -741,6 +741,7 @@ public function FieldHolder($properties = [])
                 'tr',
                 [
                     'class' => 'ss-gridfield-item ss-gridfield-no-items',
+                    'role' => 'row',
```

Comment:
```suggestion
                    // The "role" attribute should be redundant, but since it's clickable via javascript some assistive technologies may think it's an interactive element for the purposes of WCAG 2.2 A 4.1.2. We hope that adding an explicit "role" helps resolve that confusion.
                    'role' => 'row',
```
Something like this so we don't just remove it again later.

Reply:
Updated